tsconfig.json

```diff
@@ -8,6 +8,6 @@
     "esModuleInterop": true,
     "typeRoots": [ "node_modules/@types" ]
   },
-  "include": [ "./src/**/*" ],
+  "include": [ "./src/**/*", "./test/**/*" ],
```

Comment:
Are you sure this works? I've tried this before and the app will fail to build.
I tried this again locally just now and same error appears.

`Error: Cannot find module '/Users/waseem/Projects/ChuckieNorrisieTwitterBot/dist/index.js'`

Reply:
I thoght, I published a new version 🤔
You are right, this will not work. You could change the `node dist/index.js` to `node dist/src/index.js` or use a `tsconfig.spec.json` file that I have created in the nine version